### ISSUE TITLE
Static data/params on the Snippet level available to the handlebars 

### ISSUE THREAD

**Issue by marcinczeczko** [Knotx/knotx#391]
### Version used

* knot.x: 1.2.1

### Context of the issue/feature
Snippet fragment sometime should use some data that were processed during rendering of the <script> tag, by modifying HTML structure. However, since the knotx snippet is in the special HTML tag, some CMS system are not able to process the tags inside the fragment.
In such a cases, CMS could render this data (e.g. paths to image renditions into the snippet param attribute. Data from such attribute would be available in the Handlebars context, so the data can be easily rendered.
```html
<script data-knotx-knots="services,handlebars"
            data-knotx-service="some-service"
            data-knotx-service-data="{'img':'/path/image.png'}"
            type="text/knotx-snippet">
    <img src="{{_result._data.img}}"/>
</script>
```
A new attribute "data-knotx-service-data". Other approach, is to expose data from data-knotx-service-params attribute... into the handlebars context.

### Bug reproducer

None

### Steps to reproduce

None

### Appendix

None

**Comment by tomaszmichalak** [comment]
I thought about it by analyzing the isolation of nodes in the routing (Knots). My idea was to have some kind of namespace for each Knot.

Example:
```html
<script data-knotx-knots = "services, steering wheels"
    data-knotx-service = "some-service"
    data-knotx-service-params = "{'img': '/ path / image.png'}"
    data-knotx-handlebars-data = "{'img': '/ path / image.png'}"
    type = "text / knotx-snippet">
        <img src = "{{_ result._data.img}}" />
</ script>
```
Then the Service Knot sees only params within its namespace (service), the same Handlebars Knot does.

**Comment by tomaszmichalak** [comment]
In this way we can have some global params: `data-knotx-all-param1 = "some-service"`.

**Comment by Skejven** [comment]
Won't fix for #107 from 1.2.1 help this problem (change the `<script>` tag to something other).
I'm not sure about this idea... If we want to go this way we shouldn't mix `_result` with some static data that is actually not from the response.

```xml
<script data-knotx-knots="services,handlebars"
            data-knotx-service="some-service"
            data-knotx-service-const="{'img':'/path/image.png'}"
            type="text/knotx-snippet">
    <p>Hello {{_result.message}}</p>
    <img src="{{_const.img}}"/>
</script>
```
